kube/are-rc.yaml

```diff
@@ -11,14 +11,18 @@ spec:
       labels:
         name: are
     spec:
+      securityContext:
+        runAsUser: 1000
+        runAsNonRoot: true
       containers:
       - name: app
         image: quay.io/ukhomeofficedigital/are-app:v0.0.1
-        imagePullPolicy: Always
         env:
         - name: REDIS_HOST
-          value: redis.are-dev.svc.cluster.local
+          value: redis.${NAMESPACE}.svc.cluster.local
         - name: REDIS_PORT
           value: "6379"
         ports:
         - containerPort: 8080
+        securityContext:
+          readOnlyRootFilesystem: true
```

Comment:
This change make break your app depending on if the node app needs to write to the filesystem, which is something I haven't generally tested.  If this does break it, you need to add a emptydir where the runtime writes to, but feel free to seek my guidence if and when you hit this wall.

FYI: Your current build definitely writes to the filesystem in production because the build is broken and I've hacked it do run a transpile on startup.  You should rebuild it to have it's assests properly embedded into the container before it starts.